src/ir_Pioneer.h

```diff
@@ -0,0 +1,31 @@
+// Copyright 2009 Ken Shirriff
```

Comment:
TL;DR: This is a minor style/convention nitpick.

Please move these constants into the `ir_Pioneer.cpp` file. e.g. remove the header file. ;-)

Longer version:
Unless we absolutely need to, by convention "simple" protocols don't have associated header files. e.g. NEC & LG do, but only because other protocols use some of their constants etc.
Generally only the "complex" aircon protocols have them.

Having the protocol timings in the same file as the send/decode routines also helps non-programmers find the values they often need to tweak easier.

And, the super-duper lamest reason which is most important to me. It allows me to quickly scan the protocol files and identify which ones have complex A/C protocols in them. ;-)

Otherwise, yes, I'd be all for using header files. ;-)

Reply:
Done